src/client/Client.cc

```diff
@@ -6524,13 +6524,17 @@ int Client::_do_setattr(Inode *in, struct stat *attr, int mask, int uid, int gid
       mask &= ~(CEPH_SETATTR_MTIME|CEPH_SETATTR_ATIME);
     }
   }
-  if (!mask)
+  if (!mask) {
+    in->change_attr++;
     return 0;
+  }
 
 force_request:
   MetaRequest *req = new MetaRequest(CEPH_MDS_OP_SETATTR);
 
   filepath path;
+
+  in->change_attr++;
```

Comment:
I'm not sure I understand that bit. Why would it get bumped again on the remote side? My understanding is that when the client returns exclusive caps, the MDS just accepts what is in the cap message from the client as gospel. Is that not the case?